src/SGFParser.cpp

```diff
@@ -38,6 +38,81 @@
 #include "SGFTree.h"
 #include "Utils.h"
 
+std::vector<SGFParser::ELF_Data> SGFParser::chop_elf(std::string filename,
```

Comment:
If it's not actually parsing SGF then just put it into its own file?

Reply:
What about making it a member of `Training` as well? I originally started working in SGFParser.cpp because I would reuse some code there.